app/src/pages/inside/projectSettingsPageContainer/content/defectTypes/defectSubType.jsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Parser from 'html-react-parser';
+import { useIntl } from 'react-intl';
+import classNames from 'classnames/bind';
+
+import BinIcon from 'common/img/newIcons/bin-inline.svg';
+import PencilIcon from 'common/img/newIcons/pencil-inline.svg';
+import CopyIcon from 'common/img/newIcons/copy-inline.svg';
+import LocatorIcon from 'common/img/newIcons/locator-icon-inline.svg';
+import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
+
+import { withHoverableTooltip } from 'components/main/tooltips/hoverableTooltip';
+
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { defectTypeShape } from './defectTypeShape';
+import styles from './defectTypes.scss';
+
+const cx = classNames.bind(styles);
+
+const ColorMarker = ({ color }) => (
+  <span className={cx('color-marker')} style={{ backgroundColor: color }} />
+);
+
+ColorMarker.propTypes = {
+  color: PropTypes.string.isRequired,
+};
+
+const DefectLocator = withHoverableTooltip({
+  /* eslint-disable react/prop-types */
+  TooltipComponent: ({ locator }) => (
+    <>
+      <b>ID locator</b>
+      <br />
+      <div className={cx('locator')}>
+        {locator}
+        <CopyToClipboard text={locator}>
+          <i className={cx('icon', 'copy-button')}>{Parser(CopyIcon)}</i>
+        </CopyToClipboard>
+      </div>
+    </>
+  ),
+  data: {
+    placement: 'top',
+    dynamicWidth: true,
+    tooltipTriggerClass: cx('defect-type-name-tooltip-trigger'),
+    noArrow: true,
+  },
+})(() => <i className={cx('icon')}>{Parser(LocatorIcon)}</i>);
+
+DefectLocator.propTypes = {
+  locator: PropTypes.string.isRequired,
+};
+
+export const DefectSubType = ({
+  data: { longName, shortName, locator, color },
+  group,
+  isPossibleUpdateSettings,
+}) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <div className={cx('defect-type')}>
```

Comment:
Please keep naming the same to be consistent.
Your component and file name should be "defect type".